contracts/cw20-base/src/contract.rs

```diff
@@ -902,7 +911,7 @@ mod tests {
         );
 
         // run the migration
-        let env = mock_env(HumanAddr::from("admin"), &[]);
+        let (env, _info) = mock_env_info(HumanAddr::from("admin"), &[]);
```

Comment:
don't we need to add info to migrate so it compiles in wasm mode?

Reply:
Yes you are right. I did it locally but I suppose you are working on it don't want to tip toes.